app-editors/efte/efte-1.1-r1.ebuild

```diff
@@ -32,8 +32,8 @@ PATCHES=(
 
 src_configure() {
 	local mycmakeargs=(
-		$(cmake-utils_use_build gpm)
-		$(cmake-utils_use_build X)
+		-DBUILD_GPM=$(usex gpm)
```

Comment:
``
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_GPM
``

This is why you test your changes.

Reply:
Sonuavthebetch. I must have glossed over that part when checking out `efte-1.1/src/CMakeLists.txt`, it got switched to `USE_GPM` or sommat. Sec.